src/data-storage/index.ts

```diff
@@ -0,0 +1,3 @@
+import { insertTrade } from './insert-trade';

```

Comment:
Flavor: Even though it will replicate the **exact** same functionality, it's a little bit cleaner to just use:
`export * from './insert-table'`